.gitignore

```diff
@@ -50,5 +50,5 @@ qtcreator-*
 CATKIN_IGNORE
 
 # uwrt_mars_rover_hw_bridge shouldnt be added as a git submodule
```

Comment:
change comment to utils as well

Reply:
although it now resides as a part of the `uwrt_mars_rover_utils` package, the submodule and repo name is still `uwrt_mars_rover_hw_bridge`